<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotPropertiesWidget</class>
 <widget class="QWidget" name="RobotPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,1,1">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="xPlotWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="magPlotWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QWidget" name="yPlotWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <zorder>tetaPlotWidget</zorder>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QWidget" name="tetaPlotWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Show Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Show Plan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="haltButton">
       <property name="text">
        <string>Halt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="batteryChargeBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="robotIDLabel">
       <property name="text">
        <string>ID: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="isInFieldLabel">
       <property name="text">
        <string>In Field:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robotRoleLabel">
       <property name="text">
        <string>Role:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robotSkillLabel">
       <property name="text">
        <string>Skill:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
